R/panel.r

```diff
@@ -27,7 +27,7 @@ new_panel <- function() {
 # @param plot_data the default data frame
 # @return an updated panel object
 train_layout <- function(panel, facet, data, plot_data) {
```

Comment:
Getting rid of the `panel` as a code object/file name resolves the concept clash that has existed between `facet` and `panel`.